src/js/components/Login.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import GoogleLogin from 'react-google-login';
+import FontAwesome from 'react-fontawesome';
+
+
+export default class Login extends React.Component {
+  constructor() {
+    super();
+    this.responseGoogle = this.responseGoogle.bind(this);
```

Comment:
remove this from here, not necessary if you define the method as an arrow function. See comment in `src/js/components/Header.jsx` above